cli/setup-arcos.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# Copyright (c) 2020, General Electric Company and Galois, Inc.
+set -eu
+
+if ! command -v rack > /dev/null
+then
+	cat <<-END
+		ERROR: rack cli tool not found in PATH
+		
+		Installation instructions are available at
+		https://github.com/ge-high-assurance/RACK/wiki/RACK-CLI#install-dependencies
+		or locally in README.md
+		
+		If you've already installed RACK CLI, please activate your virtual environment
+		
+		macOS/Linux: source venv/bin/activate
+		Windows:     venv\\Scripts\\activate.bat
+		PowerShell:  venv\\Scripts\\Activate.ps1
+	END
+	exit 1
+fi
+
+# Boeing model & auto-generated nodegroups
+rack model import ../Boeing-Ontology/OwlModels/import.yaml
+rack nodegroups import ../nodegroups/ingestion/Boeing
+
+# GrammaTech model & auto-generated nodegroups
+rack model import ../GrammaTech-Ontology/OwlModels/import.yaml
+rack nodegroups import ../nodegroups/ingestion/GrammaTech
+
+# LM model & auto-generated nodegroups
+rack model import ../LM-Ontology/OwlModels/import.yaml
+rack nodegroups import ../nodegroups/ingestion/LM
+
+# SRI model & auto-generated nodegroups
+rack model import ../DesCert-Ontology/OwlModels/import.yaml
+rack nodegroups import ../nodegroups/ingestion/SRI
+
+# STR model & auto-generated nodegroups
+rack model import ../STR-Ontology/OwlModels/import.yaml
+rack nodegroups import ../nodegroups/ingestion/STR
+
+# load applicable standards
+# rack data import --clear ../RACK-Ontology/OwlModels/ARP-4754A.yaml     # from datagraph http://rack001/arp-475a
+# rack data import --clear ../RACK-Ontology/OwlModels/DO-330.yaml        # from datagraph http://rack001/do-330
+rack data import --clear ../RACK-Ontology/OwlModels/DO-178C.yaml         # from datagraph http://rack001/do-178c
```

Comment:
Should this be loaded by the Turnstile script, instead?

Reply:
That's a good question. @russell-d-e, when do you think DO-178C should be loaded? When setting up ARCOS or when loading Turnsile data?

Reply:
I think it should be loading by whomever is using.  So it should be loaded by Turnstile and if someone is going to related there generated evidence then should include it as well.  If there is multiple providers wanting to use it they all should include it as there ingestion package.  As long as everyone uses the same data there should not be any duplicates (as long as the nodegroups are constructed with a lookup).